js/samples/quickstart-kotlin-azfunction/app/src/main/java/com/example/immersivereadersdk/MainActivity.kt

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.example.immersivereadersdk
+
+import android.app.Activity
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.webkit.CookieManager
+import android.webkit.WebView
+import android.widget.Button
+import android.webkit.WebViewClient
+import android.widget.LinearLayout
+import android.widget.RadioButton
+import android.widget.TextView
+import com.google.gson.*
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.android.synthetic.main.activity_main.*
+import java.io.IOException
+import java.io.*
+import java.net.HttpURLConnection
+import java.net.HttpURLConnection.HTTP_OK
+import java.net.URL
+import kotlinx.coroutines.*
+import org.json.JSONObject
+import java.util.*
+
+
+class MainActivity : AppCompatActivity() {
+    private val dotEnv = dotenv {
+        directory = "/assets"
+        filename = "env"
+        ignoreIfMalformed = true
+        ignoreIfMissing = true
+    }
+
+    var tokenFrom = "local"
```

Comment:
No because it is for development purposes only, if app is deployed there is no way to have local setup for function in the mobile device.
Comment added in source code for clarification

Reply:
Is this called out in the Readme?

Reply:
added